### ISSUE TITLE
Consider Heroku Deployment for Ember Apps

### ISSUE THREAD

**Issue by soynog** [ga-wdi-boston/ember-deployment-guide#3]
I just deployed my capstone project to Heroku, and it was quite simple, and makes redeployment (as far as I can tell) much easier.

I followed the instructions to create this buildpack and deploy:
https://github.com/tonycoco/heroku-buildpack-ember-cli

And I had to follow the instructions referenced in [this issue](https://github.com/ga-wdi-boston/capstone-project/issues/141) to set up environment variables.

You can test whether or not the app will deploy successfully by seeing if Ember build runs--I was having a merge conflict that prevented this from working, but once I solved that it deployed like a charm.

Anyway point is, Heroku seems about as easy or possibly easier than gh-pages for Ember app deployment; something to consider for future cohorts.

**Comment by jrhorn424** [comment]
Thanks @soynog!

**Comment by gaand** [comment]
I believe @berziiii has some issues getting this to work.  Brian, do you have any notes from that experience?

**Comment by berziiii** [comment]
Yes I did. I didn't have as much mental capacity to dedicate to debugging as I would have liked but in following the above build pack guide, I always had an error once deployed. I also referenced the linked issue.

The build seemed to be successful, however when trying to visit the deployed client, I received an error. Again, I did not dig deep enough into the logs due to GitHub deployment being successful and painless. 

IMHO, it may have been some updates to ember/build-pack to cause deployment to fail but no real data to back - take it as you will.

Happy to assist when the time comes to address this guide again.

**Comment by BenGitsCode** [comment]
I did not make enough time to pursue this alternative, but I think we should try it to contrast and compare the pedagogical value of changing it or not.

**Comment by BenGitsCode** [comment]
Hello again. Still didn't have enough time to pursue this option (my motivation was also low with gh-pages working so well) but I'm going to try it this weekend, and record the experience here, so it can be considered for the next iterations in `lm-01` and `017`
